src/ui/components/CopyButton/CopyButton.tsx

```diff
@@ -48,12 +73,17 @@ export function CopyButton({ address, title = 'Copy Address' }: Props) {
             color: 'var(--white)',
             boxShadow: 'var(--elevation-200)',
             position: 'absolute',
-            top: 2,
-            right: 0,
-            transform: 'translateX(100%)',
             padding: '4px 8px',
             borderRadius: 4,
             whiteSpace: 'nowrap',
+            top: tooltipPosition === 'center-bottom' ? undefined : 2,
```

Comment:
I think, this can be one separate object, which can be spread with condition here

Reply:
Thought of that but spreading loses CSSProperties types for some reason, and since these are still very much hardcoded values and not a truly adaptive tooltip I don't think it's worth to bother. Once css anchors make it to stable versions we can finally implement them in the ui-kit :)